src/api/app/helpers/webui/obs_factory/application_helper.rb

```diff
@@ -8,4 +8,10 @@ def distribution_tests_url(distribution, version = nil)
     path << "&build=#{version}" if version
     path
   end
+
+  def icon_for_checks(checks)
+    return 'eye' if checks.any? { |check| check.state == 'pending' }
```

Comment:
why?
I assume rubocop would complain about it ...

Reply:
so we just review what rubocop complains?

Reply:
you mean, rubocop will complain about the if/elsif/else?

Reply:
Yo, rubocop usually prefers guard statements. Although I haven't tried it! But anyway, I don't see an advantage over if/else over the guard statement except personal preference ...

Reply:
I was thinking that the coverall would increase the method complexity, since the ```return if``` 2 times and a static string as default return value is a clear ```if elsif else``` case, but if you want to keep it, fine by me.